kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ _('{title}')|fe(title=document.title) }}</a></h1>
```

Comment:
Spot-check looks good but I'd like @jezdez, @robhudson, or @ubernostrum to double-check me that this safe use of [`fe`](https://jingo.readthedocs.org/en/latest/#jingo.helpers.fe).